frontend/src/tbpro/elements/BaseButton.vue

```diff
@@ -72,6 +77,21 @@ button:hover > .tooltip {
   }
 }
 
+.link {
+  background-color: transparent;
+  color: var(--tbpro-primary);
+  text-decoration: underline;
+  box-shadow: none;
+  border: none;
+
+  .text {
+    padding: 0;
+    user-select: all;
+    font-weight: 400;
+    line-height: 1;
+  }
+}
```

Comment:
I was thinking of moving individual styles to the alias components (PrimaryButton, SecondaryButton, LinkButton) but that'll be a wider discussion when we consolidate the components into a separate repo.

Reply:
Yes, sounds good!